services/ui-src/src/components/export/ExportedModalDrawerReportSection.tsx

```diff
@@ -6,7 +6,7 @@ import { EntityCard, ReportContext } from "components";
 import { getFormattedEntityData } from "utils";
 import { EntityShape, ModalDrawerReportPageShape } from "types";
 // verbiage
-import exportVerbiage from "verbiage/pages/export";
+import exportVerbiage from "verbiage/pages/mcpar/mcpar-export";
```

Comment:
does MLR not have export verbiage here?

Reply:
@tblackwe we do not have a Modal Drawer for MLR currently.